ui/raidboss/data/04-sb/trial/lakshmi-ex.txt

```diff
@@ -40,7 +40,7 @@ hideall "Alluring Arm"
 225.5 "The Pull Of Light" # end of castbar
 228.5 "The Path Of Light" duration 5.5 # hat -> cleave
 246.0 "Divine Desire" # end of castbar
-249.5 "--Chanchala over--"
+249.5 "--chanchala end--"
```

Comment:
If you are going to change this, can you edit (remove?) the timeline replacements as well that are using this?